src/ui/item.jsx

```diff
@@ -64,10 +65,43 @@ class Item extends React.PureComponent {
       </div>
     );
   }
+
+  tooltipContent() {
+    const { locations } = this.props;
+
+    return (
+      <div className="tooltip item-location">
+        <ul>
+          {locations.map(({ generalLocation, detailedLocation }) => (
+            <li key={`${generalLocation}-${detailedLocation}`}>
+              {generalLocation}
+              {' | '}
+              {detailedLocation}
```

Comment:
```suggestion
              {`${generalLocation} - ${detailedLocation}`}
```

Reply:
I noticed that when the location names have `-` in the name it can get a little awkward to read with just a `-` separator. Should we still go with the hyphen or use a different character?

Reply:
I guess it does make sense to use `|`. However, I'd prefer to stick to string interpolation:
```
{`${generalLocation} | ${detailedLocation}`}
```
Could you make this change both here and for sphere tracking? https://github.com/wooferzfg/tww-rando-tracker/blob/main/src/ui/sphere-tracking.jsx#L45

Reply:
Yup, I made the change to use string interpolation as well.